src/ResourceGraph/ResourceGraph/ChangeLog.md

```diff
@@ -19,6 +19,11 @@
 -->
 ## Upcoming Release
 
+## Version 0.9.0
+* Added support for the new api version with the ability to query with management group scopes using -ManagementGroup param.
+* Deprecated parameter -Include.
+* Introduced -SkipToken param and aligned max resources returned per page with server value.
+
```

Comment:
Please move change log msgs to the top of ##upcoming release. Refer to: https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/azure-powershell-developer-guide.md#change-log